scripts/enum/animation_strings.lua

```diff
@@ -0,0 +1,107 @@
+-----------------------------------
+xi = xi or {}
+
+--[[
+Looks like in other places we call them entityAnimation and in another for zone parts using a diff packet but the same index (like lamps in psoxja, geysers in dangruf wadi) uses entityVisual
+
+-----------------------------------
+mob:entityAnimationPacket(xi.animationString.STATUS_VISIBLE)
+-----------------------------------
+The base entity is a mob, and anyone in range to see the mob sees the animation.
+We do accept a second argument in this one, for target.
+An example of this is the afflictors in Beadeaux, which target the player. npc:entityAnimationPacket('main', player)
+
+-----------------------------------
+player:entityVisualPacket('saa1')
+-----------------------------------
+In this there is no mob  or npc at all.
+Tt's a part of the zone itself animating, so player is used as base entity and any player that zones in gets sent the packet to light up that part of the zone.
+So the thing animating it's really the zone itself, sort of.
+(There is technically "an object"  or "entity" but its not ID'd like an NPC, and technically speaking a zone dat is laid out like NPCs are part of it anyway)
+
+-----------------------------------
+SendEntityVisualPacket(ID.npc.GEYSER_OFFSET, 'kkj2')
+-----------------------------------
+This one is a luautils binding, sending a packet to the entire zone (of whoever is in visual range of the ID specified) on demand.
+
+This are FOURCC codes.
+For most things you can get away with using the wrong one, and they still just work.
+--]]
+
+xi.animationString =
+{
+    -- Physical
+    ATTACK_1                  = 'ati0',
+    ATTACK_2                  = 'ati1',
+    ATTACK_3                  = 'ati2',
+    CRITICAL_HIT              = 'chit',
+
+    -- Ranged start
+    RANGED_SINGING_START      = 'lc00',
+    RANGED_WIND_INSTR_START   = 'lc01',
+    RANGED_STRING_INSTR_START = 'lc02',
+    RANGED_GUN_START          = 'lc03',
+    RANGED_BOOMERANG_START    = 'lc04',
+    RANGED_BOW_START          = 'lc06',
+
+    -- Ranged stop
+    RANGED_SINGING_STOP       = 'ls00',
+    RANGED_WIND_INSTR_STOP    = 'ls01',
+    RANGED_STRING_INSTR_STOP  = 'ls02',
+    RANGED_GUN_STOP           = 'ls03',
+    RANGED_BOOMERANG_STOP     = 'ls04',
+    RANGED_BOW_STOP           = 'ls06',
+
+    -- Casting start
+    CAST_BLACK_MAGIC_START    = 'cabk',
+    CAST_BLUE_MAGIC_START     = 'cabl',
+    CAST_ITEM_START           = 'cait',
+    CAST_NINJA_START          = 'canj',
+    CAST_SUMMONER_START       = 'casm',
+    CAST_SONG_START           = 'caso',
+    CAST_WHITE_MAGIC_START    = 'cawh',
+
+    -- Casting finish
+    CAST_BLACK_MAGIC_STOP     = 'shbk',
+    CAST_BLUE_MAGIC_STOP      = 'shbl',
+    CAST_ITEM_STOP            = 'shit',
+    CAST_NINJA_STOP           = 'shnj',
+    CAST_SUMMONER_STOP        = 'shsm',
+    CAST_SONG_STOP            = 'shso',
+    CAST_WHITE_MAGIC_STOP     = 'shwh',
+
+    -- Actions
+    JUMP_1                    = 'jmp0',
```

Comment:
I think its less confusing to stick to the same number SE used, like you did with the specials where you started at 0